src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int numberBall = 100;
+    private Random random;
+    private ColorSupplier colorSupplier;
+
+    public ColorSupplier getColorSupplier() {
+        return colorSupplier;
+    }
+
+    public void setColorSupplier(ColorSupplier colorSupplier) {
+        this.colorSupplier = colorSupplier;
+    }
+
+    public Ball getRandomBall() {
+        colorSupplier = new ColorSupplier();
+        random = new Random();
```

Comment:
Assign a values to random and colorSupplier immediately or in the constructor

Reply:
Not fixed.
Initialize either through the constructor or in a variable directly:
private Random random = new Random();
private ColorSupplier colorSupplier = new ColorSupplier();